normalizer/app/streamlit_app.py

```
@@ -203,12 +300,18 @@ def progress_callback(idx, total):
             progress_bar.progress(1.0)
             status_text.text("完了")
             info_placeholder.empty()
-            msg = "処理完了！出力データをダウンロードできます"
+            
+            # 「緯度経度は怪しい」のカウント
+            total_rows = len(out_df)
+            suspicious_count = 0
+            if "note" in out_df.columns:
+                suspicious_count = sum(1 for note in out_df["note"] if "緯度経度は怪しい" in str(note))
+            
+            msg = f"処理完了！{total_rows}件中{suspicious_count}件({suspicious_count/total_rows*100:.1f}%)が緯度経度は怪しいとなりました。出力データをダウンロードできます"
```

Ruff (0.14.13)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)